### DISCUSSION TITLE
clientKey production not working

### DISCUSSION THREAD

**Discussion by shailesh941** [Unleash/unleash#3772] [Q&A] [answered]
default:development.xyzzzz,

default:production.xyzzzz

**Comment by shailesh941** [comment]
default:development.xyzzzz this key on not working production toggle, please suggest production toggle clientKey

**Reply by pransh15** [reply]
I’m sorry @shailesh941, but I’m not familiar with the specific context you’re referring to. Can you provide more information or clarify your question?

**Reply by gardleopard** [reply, marked as answer]
@shailesh941 to create a token for production you can look at this documentation: https://docs.getunleash.io/how-to/how-to-manage-public-invite-tokens#creating-a-token

If you want to learn more about tokens you can have a look here: https://docs.getunleash.io/reference/api-tokens-and-client-keys
